src/Tables.jl

```diff
@@ -8,29 +8,128 @@ if !hasmethod(getproperty, Tuple{Tuple, Int})
     Base.getproperty(t::Tuple, i::Int) = t[i]
 end
 
-"Abstract row type with a simple required interface: row values are accessible via `getproperty(row, field)`; for example, a NamedTuple like `nt = (a=1, b=2, c=3)` can access its value for `a` like `nt.a` which turns into a call to the function `getproperty(nt, :a)`"
-abstract type Row end
+"""
+    Tables.AbstractColumns
```

Comment:
Can we highlight explict that not all column types inherit from AbstractColumn ?
And that it thus should not be depended on for dispatch. 
That it is just for convieinve of source authors (not sink authors)
(And same for Row)

Reply:
Great call out.